jenkins/bisect.jpl

```diff
@@ -530,6 +546,10 @@ def bisection(kci_core, kdir, checks) {
     def good = null
     def bad = null
 
+    def ref_kernel_url = params.REF_KERNEL_URL
```

Comment:
I think these aren't used anywhere.

Reply:
Moved to findMergeBased where they should be defined.